benchmarks/cosmos-exim/main.go

```diff
@@ -0,0 +1,196 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/tendermint/iavl"
+	tmdb "github.com/tendermint/tm-db"
+)
+
+// stores is the list of stores in the CosmosHub database
+// FIXME would be nice to autodetect this
+var stores = []string{
```

Comment:
In lieu of auto-detecting this, perhaps it can be pushed to a config in a future PR.